bolt/test/binary-analysis/AArch64/lit.local.cfg

```diff
@@ -1,7 +1,8 @@
 if "AArch64" not in config.root.targets:
     config.unsupported = True
 
-flags = "--target=aarch64-linux-gnu -nostartfiles -nostdlib -ffreestanding"
+# -Wl,--no-relax prevents converting ADRP+ADD pairs into NOP+ADR.
+flags = "--target=aarch64-linux-gnu -nostartfiles -nostdlib -ffreestanding -Wl,--no-relax"
```

Comment:
I'm not entirely sure, but maybe we should only add `-Wl,--no-relax` to test case gs-pauth-address-materialization.s (and other test cases that require it in the future)?
If we add too many special flags to the default flags for all the binary analysis test cases, we might end up running all test cases accidentally under some flags that are too far removed from the typical flags used to build production software?